lib/foreman/renderer.rb

```diff
@@ -26,6 +26,18 @@ def template_logger
       @template_logger ||= Foreman::Logging.logger('templates')
     end
 
+    def host_info(*path)
```

Comment:
Can you choose a better name for this, or even remove it if unnecessary? The model method isn't well-named, it's specifically the Puppet ENC output, not "info".

I don't really see much use for the Puppet ENC output, I'd prefer to see more specific information like `host_param`, so if you're trying to provide a list of classes to the template for example, a helper to return host classes would be better.